src/modules/GroupChannel/components/MessageList/getMessagePartsInfo.ts

```diff
@@ -30,11 +30,12 @@ export const getMessagePartsInfo = ({
   currentMessage = null,
   currentChannel = null,
   replyType = '',
-}: GetMessagePartsInfoProps): OutPuts => {
+  currentUserId = undefined
+}: GetMessagePartsInfoProps & { currentUserId?: string }): OutPuts => {
```

Comment:
currentUserId will be undefined by default anyway right? Any reason to explicitly given it a default of undefined here?